src/app/clusters/icd-management-server/icd-management-server.cpp

```diff
@@ -64,6 +65,18 @@ class IcdManagementAttributeAccess : public AttributeAccessInterface
     CHIP_ERROR ReadIdleModeDuration(EndpointId endpoint, AttributeValueEncoder & encoder);
     CHIP_ERROR ReadActiveModeDuration(EndpointId endpoint, AttributeValueEncoder & encoder);
     CHIP_ERROR ReadActiveModeThreshold(EndpointId endpoint, AttributeValueEncoder & encoder);
+    CHIP_ERROR ReadFeatureMap(EndpointId endpoint, AttributeValueEncoder & encoder)
+    {
+        return encoder.Encode(mICDConfigurationData->GetFeatureMap());
+    }
+
+#if CHIP_CONFIG_ENABLE_ICD_LIT
```

Comment:
What happens in a configuration where CHIP_CONFIG_ENABLE_ICD_LIT is false but CHIP_CONFIG_ENABLE_ICD_CIP is true? The spec explicitly allows such a configuration, but it looks like this code would not build/work right in it?

Reply:
Moved the CIP and UAT out of the LIT

> We should add such a configuration to our CI, probably, to catch future issues.
 
yes, looks like our CI doesn't test with CIP/UAT enabled but LITS disabled. Raise #37066 to follow it.